### PR TITLE
Add Basic footer layout and links

### LINKED ISSUE #28: Add some navigation to the footer
## User story
As a person using the site I need an obvious way to navigate around the site, so that I can easily find what I am looking for.

## Acceptance criteria
- [ ] I can see a list of links in the footer with appropriate headings (list TBC below)
- [ ] (TBC) I can tell which link I am currently on (@undividual do we want some "active state" for the footer menu links?)

## Implementation notes & questions
Probably worth adding the responsive layout for footer "columns" with this if that hasn't been done in #26

## List of links
Find out more
- [x] [Visit The Wildlife Trusts main website](https://www.wildlifetrusts.org)

Explore the library
- [x] Find a guide (links to the search page - defaults to alphabetical list)

Get involved
- [x] [Find your local trust](https://www.wildlifetrusts.org/wildlife-trusts)

## Add these when the pages are ready. #51 
 - [ ] Find out more about this project 
 - [ ] How to use this website
 - [ ] Full sitemap
 - [ ] Submit your own Case study

## Implementation plan
- [x] Create a styled container component with some flex to display columns appropriately across different screen sizes
- [x] Add list of links, urls and titles to keys
- [x] Create styled column component & implement

### PR DESCRIPTION
**Pull request by r-ferrier**
Fixes #28 

## Description

- Adds a basic responsive footer column layout and relevant links/ placeholders

@geeksforsocialchange/developers

### CODE REVIEW

**Comment by r-ferrier** [comment]
package.lock has wormed its way in here 🪱 but pls ignore, I will remove it!

File: src/Theme/FooterTemplate.elm [hunk 1]
```diff
@@ -1,21 +1,95 @@
...
+    [ { title = FooterTitleColumnA
+      , links =
+            [ { text = FooterVisitWebsiteText, href = FooterVisitWebsiteLink }
+            , { text = FooterFindOutMoreText, href = FooterFindOutMoreLink }
+            , { text = FooterHowToText, href = FooterHowToLink }
+            ]
+      }
+    , { title = FooterTitleColumnB
+      , links =
+            [ { text = FooterExploreAlphabeticallyText, href = FooterExploreAlphabeticallyText }
```
Comment (aaaaargZombies) [inline src/Theme/FooterTemplate.elm:47]:
```suggestion
            [ { text = FooterExploreAlphabeticallyText, href = FooterExploreAlphabeticallyLink }
```

**Review by aaaaargZombies — APPROVED** [review]
Looking good, really enjoying the emojis.

enjoy merging :o

**Comment by katjam** [comment]
> > package.lock has wormed its way in here worm but pls ignore, I will remove it!
> 
> I think its fine to leave it in, we need to track changes but we don't need to read it.

We do need to figure out why it's changing though. Unless you are changing the packages or using a different version of node, it shouldn't change. Is your build environment respecting the `.nvmrc` ? You should have it set up to `nvm use`

**Comment by katjam** [comment]
I'm doing some work over the weekend due to short notice on commitments next week & we're in Alpha, what could go wrong? So, normally I would not do this, but fixing conflicts and merging on your behalf to get things up to date.

**Comment by katjam** [comment]
I'm also removing all the dead links. Technically, since we are using [cCc] these would not get through, but I deliberately made the AC so we don;t end up with something broken after any PR.

It's a good discipline to practice when you build knowing you won't have time to build everything.

File: src/Theme/FooterTemplate.elm [hunk 2]
```diff
@@ -1,21 +1,95 @@
...
+view : Language -> Html msg
+view language =
     footer [ css [ footerStyle ] ]
-        [ h2
-            []
-            [ text "[cCc] Footer"
+        [ div [ css [ navigationContainerStyle ] ]
+            (List.map
+                (\column -> navigationColumn column language)
+                footerTitles
```
[the remark this answers is not in the document]
Comment (katjam) [reply src/Theme/FooterTemplate.elm:16]: I'll change to footerNavLinkContent

**Comment by katjam** [comment]
@aaaaargZombies @r-ferrier No need to look over the weekend... but I'm at a conference all day tomorrow, so thought I'd let this sit for review.

I'll merge on Sunday before I start in on other stuff for a clean slate. I mostly kept your work, just had a small re-jig. :)

**Pull request merged**